e2e/framework/clients/k8s_rook_block.go

```diff
@@ -3,29 +3,30 @@ package clients
 import (
```

Comment:
I am going to rename all the file and struct names and remove redundant 'Rook' .